packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -160,6 +160,7 @@ export type TrackListItemProps = {
   contextPlaylistId?: ID
   index: number
   isReorderable?: boolean
+  isAlbumPage?: boolean
```

Comment:
nit/optional, maybe change name of this prop to `showViewAlbumButton` or something like that  to keep the component isolated

Reply:
Oh good catch! I think I just lost track of the layers